CLAUDE.md

```
@@ -1,69 +1,7 @@
-# CLAUDE.md
+Digests is a RSS Reader and Podcast Player built with React. It is designed to be simple, fast, and easy to use. It supports both RSS and Atom feeds, and it can also play podcasts.
```

markdownlint-cli2 (0.20.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)